promgen/templates/promgen/service_detail.html

```diff
@@ -8,15 +8,53 @@
 
 {% block content %}
 
-<div class="page-header" v-pre>
-  <h1>Service: {{ service.name }}</h1>
-  {% if service.owner %}
-  <p>{% trans 'Contact' %}: {{service.owner.username}}</p>
-  {% endif %}
+<div class="page-header promgen-flex-space-between-center">
+  <div>
+    <h1>Service: {{ service.name }}</h1>
+    {% if service.owner %}
+    <p>{% translate 'Contact' %}: {{service.owner.username}}</p>
+    {% endif %}
+  </div>
+  
+
+  {% include "promgen/service_action_button_group.html" %}
+
 </div>
 
 {% breadcrumb service %}
 
-{% include "promgen/service_block.html" %}
+<ul class="nav nav-tabs mb-5" role="tablist">
+  <li role="presentation" class="active"><a href="#project" data-toggle="tab">Project</a></li>
+  <li role="presentation"><a href="#rule" data-toggle="tab">Rule</a></li>
+  <li role="presentation"><a href="#notifier" data-toggle="tab">Notifier</a></li>
+</ul>
```

Comment:
I'm wondering if we should use plural form here.

- Projects
- Rules
- Notifiers

We have probably not been particularly consistent across the project.

Reply:
I agree we use plural form here. I notice in the page hear, we also use plural form such as `Services` `Rules` as title